prime-router/metadata/fhir_mapping/hl7/codesystem/ExtensionUrlMapping.yml

```diff
@@ -295,4 +295,46 @@
   url: "https://reportstream.cdc.gov/fhir/StructureDefinition/identifier-namespace-id"
 # Universal Namespace ID
 - id: "identifier-universal-id"
-  url: "https://reportstream.cdc.gov/fhir/StructureDefinition/identifier-universal-id"
\ No newline at end of file
+  url: "https://reportstream.cdc.gov/fhir/StructureDefinition/identifier-universal-id"
+# Data Type Absent Reason
+- id: "data-absent-reason"
+  url: "https://reportstream.cdc.gov/fhir/StructureDefinition/data-absent-reason"
+# Contact Point Country
+- id: "contactpoint-country"
+  url: "http://hl7.org/fhir/StructureDefinition/contactpoint-country"
+# Contact Point Area
+- id: "contactpoint-area"
+  url: "http://hl7.org/fhir/StructureDefinition/contactpoint-area"
+# Contact Point Local
+- id: "contactpoint-local"
+  url: "http://hl7.org/fhir/StructureDefinition/contactpoint-local"
+# Contact Point Extension
+- id: "contactpoint-extension"
+  url: "http://hl7.org/fhir/StructureDefinition/contactpoint-extension"
+# Telecom Equipment Type
+- id: "telecom-equipment-type"
+  url: "https://reportstream.cdc.gov/fhir/StructureDefinition/telecom-equipment-type"
+# Telecom Use Code
+- id: "telecom-use-code"
+  url: "https://reportstream.cdc.gov/fhir/StructureDefinition/telecom-use-code"
+# HL7 Use
+- id: "hl7-use"
+  url: "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7-use"
+# Telephone Number
+- id: "telephone-number"
+  url: "https://reportstream.cdc.gov/fhir/StructureDefinition/telephone-number"
+# Communication Address
+- id: "communication-address"
+  url: "https://reportstream.cdc.gov/fhir/StructureDefinition/communication-address"
+# Unformatted Telephone Number
+- id: "unformatted-telephone-number"
+  url: "https://reportstream.cdc.gov/fhir/StructureDefinition/unformatted-telephone-number"
+# Local Number
+- id: "local-number"
+  url: "https://reportstream.cdc.gov/fhir/StructureDefinition/local-number"
+# Any Text
+- id: "any-text"
+  url: "https://reportstream.cdc.gov/fhir/StructureDefinition/any-text"
+# HL7 v2 Name
+- id: "hl7v2Name"
+  url: "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Name"
```

Comment:
The majority of these extensions are just to hold the value in an HL7 field so that the value is not lost or so that the correct value in a list can be identified without the need to reference an index